ref/license-information.md

```diff
@@ -0,0 +1,8 @@
+Anbox Cloud is built using multiple open source software components. This guide lists those components and where to find their license information.
+
+| Software components | Where to find their licenses |
+|--|--|
+| LXD images | Within the LXD image, <br/>`/usr/share/licenses` and `/opt/mesa/share/licenses`<br/> The copyright information of Anbox Cloud is available at `/usr/share/doc/*/copyright`. |
+| Android | In the Android container, <br/> **Settings** > **About emulated device** > **Legal information** |
+| Snaps | On the host system,<br/>`/snap/$snap-name/current/` </br>Replace $snap-name with the name of the Snap.|
```

Comment:
I'd avoid using env variable but the angle brackets in the above sentence for clarity.
E.g.
`/snap/<SNAP_NAME>/current/`
(Let's align with what https://snapcraft.io/docs/environment-variables#heading--name doc suggests)

Also this's for the regular deployment. When it comes to appliance, people have to enter one lxd instance to find that out. I'm not quite sure if it's worth mentioning that in the doc though